docs/UserGuide.md

```diff
@@ -612,27 +656,30 @@ If a parameter is expected only once in the command, but you specified it multip
 
 ### `listshortcut`: List all shortcuts
 
-**Purpose**: Lists all shortcuts in the existing shortcut library.
+**Purpose**: Lists all shortcuts in the existing shortcut library in case you want to view the shortcuts that you have previously
+created.
 
 **Format**: `listshortcut`
 
 [Return to Table of Contents](#table-of-contents)
 <br><br>
 
 
-### `clearshortcut`: Clear all shortcuts
+### `clearshortcut`: Delete all shortcuts
 
-**Purpose**: Clears all shortcuts in the existing shortcut library.
+**Purpose**: Delets all shortcuts in the existing shortcut library.
```

Comment:
I think theres a typo here for "Deletes".

Reply:
I think so too. Haha